### PR TITLE
fix: fix some bugs

### LINKED ISSUE #3733: [Refactor][config-ui] pass the param `blueprint_id` in `/api/pipelines`
## What and why to refactor

d14bd8dc-2995-4c1a-9254-2c6b455523cc

Now this page requests `/api/pipelines` without the param `blueprint_id`. It requests all pipeline..

## Describe the solution you'd like
add `blueprint_id` to request the pipelines in this blueprint.

### PR DESCRIPTION
**Pull request by likyh**
# Summary

summary write at comments

### Does this close any open issues?
Closes #3733 
Closes https://github.com/apache/incubator-devlake/issues/3640
Closes https://github.com/apache/incubator-devlake/issues/3522
Closes https://github.com/apache/incubator-devlake/issues/3490

### CODE REVIEW

File: config-ui/src/hooks/usePipelineManager.jsx [hunk 1]
```diff
@@ -184,18 +181,18 @@ function usePipelineManager(
     }
   }, [])
 
-  const fetchAllPipelines = useCallback((status = null, fetchTimeout = 500) => {
+  const fetchAllPipelines = useCallback((blueprintId, status = null, fetchTimeout = 500) => {
```
Comment (likyh) [inline config-ui/src/hooks/usePipelineManager.jsx:184]: related to #3733

File: config-ui/src/hooks/usePipelineManager.jsx [hunk 2]
```diff
@@ -131,9 +131,10 @@ function usePipelineManager(
     }
   }, [])
 
-  const fetchPipeline = useCallback((pipelineID, refresh = false) => {
```
Comment (likyh) [inline config-ui/src/hooks/usePipelineManager.jsx:134]: `refresh` hasn't been used and I don't know what it means. so it has been deleted.

File: config-ui/src/components/pipelines/StageLane.jsx [hunk 3]
```diff
@@ -130,7 +130,6 @@ const StageLane = (props) => {
 
   useEffect(() => {
     setActiveStage(stages[sK])
-    console.log('>> ACTIVE STAGE LANE', stages[sK])
```
Comment (likyh) [inline config-ui/src/components/pipelines/StageLane.jsx:133]: related to https://github.com/apache/incubator-devlake/issues/3490

File: config-ui/src/pages/blueprints/blueprint-detail.jsx [hunk 4]
```diff
@@ -409,40 +335,49 @@ const BlueprintDetail = (props) => {
...
-        fetchPipeline(activePipeline?.id)
-        fetchAllPipelines()
+    if (activePipeline?.id) {
+      if (autoRefresh) {
+        console.log('>> ACTIVITY POLLING ENABLED!')
+        pollInterval.current = setInterval(() => {
+          fetchPipeline(activePipeline?.id)
+          // setLastPipeline(activePipeline)
+        }, pollTimer)
+        return () => {
```
Comment (likyh) [inline config-ui/src/pages/blueprints/blueprint-detail.jsx:373]: related to https://github.com/apache/incubator-devlake/issues/3640.
add a return func to cancel `setInterval`.

File: config-ui/src/pages/blueprints/blueprint-detail.jsx [hunk 5]
```diff
@@ -409,40 +335,49 @@ const BlueprintDetail = (props) => {
       )
       setCurrentRun((cR) => ({
         ...cR,
-        startedAt: activePipeline?.beganAt
-          ? dayjs(activePipeline?.beganAt).format('L LTS')
+        id: activePipeline.id,
```
Comment (likyh) [inline config-ui/src/pages/blueprints/blueprint-detail.jsx:338]: related to https://github.com/apache/incubator-devlake/issues/3522.
delete `lastPipeline` and replace with `activePipeline`

File: config-ui/src/components/pipelines/StageLane.jsx (hunk 3 above)
Comment (mintsweet) [reply config-ui/src/components/pipelines/StageLane.jsx:133]: nice job.

**Review by mintsweet — APPROVED** [review]
LGTM.

**Review by klesh — APPROVED** [review]

**Pull request merged**